apps/root/src/services/earnService.ts

```diff
@@ -54,7 +55,8 @@ export class EarnService extends EventsManager<EarnServiceData> {
   }
 
   set strategiesParameters(strategiesParameters) {
-    this.serviceData = { ...this.serviceData, strategiesParameters };
+    // Usefull for updating nested objects
+    this.serviceData = { ...this.serviceData, strategiesParameters: cloneDeep(strategiesParameters) };
```

Comment:
wait why do we actually need to cloneDeep here?, the strategiesParameter is going to be another reference, and we are updating the reference of this.serviceData so it should be a new object triggering the useServiceData